core/src/main/java/org/geysermc/geyser/registry/CommonRegistries.java

```diff
@@ -0,0 +1,152 @@
+import org.geysermc.geyser.registry.provider.ProviderSupplier;
+import org.geysermc.geyser.registry.type.ItemMappings;
+import org.geysermc.geyser.registry.type.ParticleMapping;
+import org.geysermc.geyser.registry.type.SoundMapping;
+import org.geysermc.geyser.translator.level.block.entity.BlockEntityTranslator;
+import org.geysermc.geyser.translator.level.event.LevelEventTranslator;
+import org.geysermc.geyser.translator.sound.SoundInteractionTranslator;
+import org.geysermc.geyser.translator.sound.SoundTranslator;
+import org.geysermc.mcprotocollib.network.packet.Packet;
+import org.geysermc.mcprotocollib.protocol.data.game.entity.type.EntityType;
+import org.geysermc.mcprotocollib.protocol.data.game.level.block.BlockEntityType;
+import org.geysermc.mcprotocollib.protocol.data.game.level.event.LevelEvent;
+import org.geysermc.mcprotocollib.protocol.data.game.level.particle.ParticleType;
+import org.geysermc.mcprotocollib.protocol.data.game.recipe.RecipeType;
+
+public interface CommonRegistries {
+    /**
+     * A registry holding all the providers.
+     * This has to be initialized first to allow extensions to access providers during other registry events.
+     */
+    SimpleMappedRegistry<Class<?>, ProviderSupplier> providers();
+
+    /**
+     * A registry holding a CompoundTag of the known entity identifiers.
+     */
+    SimpleRegistry<NbtMap> bedrockEntityIdentifiers();
+    /**
+     * A map containing all Java entity identifiers and their respective Geyser definitions
+     */
+    SimpleMappedRegistry<String, EntityDefinition<?>> javaEntityIdentifiers();
+
+    /**
+     * A map containing all entity types and their respective Geyser definitions
+     */
+    SimpleMappedRegistry<EntityType, EntityDefinition<?>> entityDefinitions();
+    /**
+     * A registry holding a list of all the known entity properties to be sent to the client after start game.
+     */
+    SimpleRegistry<Set<NbtMap>> bedrockEntityProperties();
+
+    /**
+     * A registry holding a CompoundTag of all the known biomes.
+     */
+    SimpleRegistry<NbtMap> biomesNbt();
+    /**
+     * A mapped registry which stores Java biome identifiers and their Bedrock biome identifier.
+     */
+    SimpleRegistry<Object2IntMap<String>> biomeIdentifiers();
+
+    /**
+     * A mapped registry which stores a block entity identifier to its {@link BlockEntityTranslator}.
+     */
+    SimpleMappedRegistry<BlockEntityType, BlockEntityTranslator> blockEntities();
+
+    /**
+     * A registry containing all the Bedrock packet translators.
+     */
+    PacketTranslatorRegistry<BedrockPacket> bedrockPacketTranslators();
+    /**
+     * A registry containing all the Java packet translators.
+     */
+    PacketTranslatorRegistry<Packet> javaPacketTranslators();
+
+    /**
+     * A registry containing all Java items ordered by their network ID.
+     */
+    ListRegistry<Item> javaItems();
+    SimpleMappedRegistry<String, Item> javaItemIdentifiers();
```

Comment:
missing javadocs, method spacing

Reply:
The current Registries is also missing a javadoc, and the method spacing is done for easier reading (clustering similar registries)

Reply:
seems like a good opportunity to add a javadoc, just for consistency :p